go/caddy/pfpki/ocsp/oscp.go

```diff
@@ -0,0 +1,747 @@
+// Copyright 2013 The Go Authors. All rights reserved.
```

Comment:
Why are we adding this code to our own repo ?

Reply:
It's a custom version (parseRequest)